app/models/presenters/batch_submenu_presenter.rb

```diff
@@ -56,6 +56,19 @@ def build_submenu
       add_submenu_option pluralize(@batch.comments.size, 'comment'), batch_comments_path(@batch)
       load_pipeline_options
       add_submenu_option 'NPG run data', "#{configatron.run_data_by_batch_id_url}#{@batch.id}"
+      return unless aviti_run_manifest?
+      add_submenu_option 'Download Sample Sheet', id: @batch.id, controller: :batches, action: :generate_sample_sheet
```

Comment:
Note: the generate_sample_sheet action added to the batches controller is Element Aviti specific. generate_sample_sheet works with AvitiSampleSheet::SampleSheetGenerator only at the moment.